src/ripple/nodestore/impl/DatabaseShardImp.cpp

```diff
@@ -36,6 +37,37 @@
 #include <sys/statvfs.h>
 #endif
 
+// A lightweight RAII struct that runs the provided
+// function continuously in a new thread, and joins
+// the thread on destruction. Used for sweeping caches
+// during database import.
+template <typename F>
+struct ThreadWrapper
```

Comment:
This struct can be eliminated, see comments on sweep calls below.

Reply:
Done